main.go

```diff
@@ -5,13 +5,13 @@ import (
 	"net/http"
 	"strconv"
 
+	"go.uber.org/zap"
```

Comment:
Don't reorder the imports.

Reply:
I think this is probably visual studio code doing it, unless I forgot I did this myself. :)